from error.BaseException import BaseException


class NotFoundException(BaseException):
    pass
